api/app/Models/Article.php

```diff
@@ -141,4 +155,14 @@ public function metas()
     {
         return $this->hasMany(ArticleMeta::class, 'article_id', 'article_id');
     }
+
+    /**
+     * Get comment relationship.
+     *
+     * @return ArticleComment
+     */
+    public function comments()
+    {
+        return (new ArticleComment)->setArticle($this);
+    }
```

Comment:
Does this relationship work with the nested entities feature? We need to be able to do the following request
```
GET /articles/{articleId}
With-Nested: comments
```